weddineo-core/src/main/java/com/weddineo/exception/dto/WeddineoApiErrorDTO.java

```diff
@@ -0,0 +1,36 @@
+package com.weddineo.exception.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Data;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.context.request.ServletWebRequest;
+import org.springframework.web.context.request.WebRequest;
+
+import java.time.LocalDateTime;
+
+@Data
+public class WeddineoApiErrorDTO {
+    private final HttpStatus status;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy hh:mm:ss")
+    private final LocalDateTime timestamp = LocalDateTime.now();
+    private final String path;
+    private final String message;
+    private final String debugMessage;
+
+    public WeddineoApiErrorDTO(HttpStatus status, Throwable exception, WebRequest request) {
+        this.status = status;
+        this.message = "Unexpected error";
+        this.debugMessage = exception.getLocalizedMessage();
```

Comment:
I am not sure if returning exception message is a good practise especially on production. This constructor is ok, but you should create one that does not contain exception message - we could refactor in future global exception handler to not return exception message in production profile.

Reply:
Sorry, for no response. I created that constructor cause I took pattern from class which I extended, but of course, you have right. I will create new constructor without that message.